components/private_channel/browser/BUILD.gn

```diff
@@ -0,0 +1,29 @@
+import("//brave/build/config.gni")
+import("//brave/components/private_channel/buildflags.gni")
+
+source_set("browser") {
+    public_deps = []
+
+    sources = [
+        "request_utils.h",
+        "request_utils.cc",
+        "private_channel_service.h",
+        "private_channel_service.h",
+        "private_channel_service.cc",
+        "static_values.h",
+    ]
+
+    deps = [
+        "//base",
+        "//brave/common",
```

Comment:
this is a layering violation, components cannot depend on `//brave/common`, `//brave/renderer`, `//brave/browser`, `//brave/utility` or `//chrome/*`